mcs/class/System/ReferenceSources/EnvironmentHelpers.cs

```diff
@@ -0,0 +1,14 @@
+
+namespace System
+{
+	internal static class EnvironmentHelpers
+	{
+		internal static bool IsWindowsVistaOrAbove()
+		{
+			// Method should match the logic of the internal Environment.IsWindowsVista property in mscorlib
+			// If this method turns out to be heavily used we might want to cache at least part of this value.
+			OperatingSystem os = Environment.OSVersion;
+			return os.Platform == PlatformID.Win32NT && os.Version.Major >= 6;
```

Comment:
we can hardcode `true` here since Mono is Vista+ since some time now (and I incidentally have a bug assigned to make this clear on the website :smile: )